### ISSUE TITLE
GH: bug and enhancement templates

### ISSUE THREAD

**Issue by dvorka** [dvorka/mindforger#1394]
GitHub templates.

**Comment by dvorka** [comment]
Fixed in f0ce2dd755ee6cca2a65807ab3429c6e3c64daa3

**Comment by akinokonomi** [comment]
https://github.com/dvorka/mindforger/blob/f0ce2dd755ee6cca2a65807ab3429c6e3c64daa3/.github/ISSUE_TEMPLATE/feature-or-enhancement.md?plain=1#L10

```diff
- **Enhacement / feature description:**
+ **Enhancement / feature description:**
```
